pkgs/misc/emulators/dolphin-emu/default.nix

```diff
@@ -28,7 +28,7 @@
 , curl
 , lzo
 , sfml
-, libpulseaudio ? null
+, libpulseaudio ? libpulseaudio
```

Comment:
`null` is only a fallback value – unless explicitly overridden, `libpulseaudio` will contain the library. So this patch does not really do anything.

Reply:
gotcha. For some reason my first installation of dolphin-emu wasn't finding any audio backend, so I rebuild it with that modification and it finally found audio backends including pulse... i'm not having that issue anymore though.